projects/element-ng/tabs-next/si-tab-next.component.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright Siemens 2016 - 2025.
+ * SPDX-License-Identifier: MIT
+ */
+import { NgClass } from '@angular/common';
+import { ChangeDetectionStrategy, Component, model, OnDestroy } from '@angular/core';
+import { SiIconNextComponent } from '@siemens/element-ng/icon';
+import { SiTranslateModule } from '@siemens/element-translate-ng/translate';
+
+import { SiTabNextBaseComponent } from './si-tab-next-base.component';
+
+@Component({
+  selector: 'si-tab-next',
+  templateUrl: './si-tab-next.component.html',
+  styleUrl: './si-tab-next.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  imports: [NgClass, SiIconNextComponent, SiTranslateModule],
+  host: {
+    '[class.active]': 'active()',
+    '[attr.aria-selected]': 'active()',
+    '(click)': 'selectTab()',
+    '(keydown.enter)': 'selectTab()'
+  }
+})
+export class SiTabNextComponent
+  extends SiTabNextBaseComponent<SiTabNextComponent>
+  implements OnDestroy
+{
+  /**
+   * Whether the tab is active or not.
+   * If set to `true`, the tab will be selected and its content will be displayed.
+   * @defaultValue false
+   * */
+  override readonly active = model<boolean>(false);
+  override activateTab(): void {
```

Comment:
Can we drop this methode and just make active.set(true);

I guess you can start using effect and linkedSignal since we now know that they are unchanged in v20